app/peopledepot/settings.py

```diff
@@ -116,8 +116,8 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": os.environ.get("SQL_ENGINE", "django.db.backends.sqlite3"),
-        "NAME": os.environ.get("SQL_DATABASE", BASE_DIR / "db.sqlite3"),
+        "ENGINE": os.environ.get("SQL_ENGINE", "django.db.backends.postgresql"),
```

Comment:
Please revert the changes to this file. We don't really want to change the behavior of defaulting to sqlite.

Just a note to clarify how these lines work: Notice the first arguments to these `os.environ.get` functions correspond to the env variable names in the `.env.docker.example` file. Those would've needed to change to something like "POSTGRES_USER" if we were to switch everything to the postgres variable names. The 2nd arguments are the default values to return in case those variables are not present.